bin/aursync

```diff
@@ -33,6 +37,13 @@ clone() {
                 warning "remote $remote does not match $repo"
             fi
 
+            # Update the config to get the newest pubkey. Is this smart or not efficient enough?
```

Comment:
It's something like 0.003s for me... should be negligable compared to the other stuff.